### PR TITLE
Orange prisoner ids now show up in wallets

### PR DESCRIPTION
**Pull request by ChangelingRain**

### CODE REVIEW

**Comment by Bawhoppen** [comment]
while you're at it how about the cap ID, and head ids might not either

**Comment by ChangelingRain** [comment]
those already show up nerd

**Comment by Bawhoppen** [comment]
cap didn't last time I checked

**Comment by ChangelingRain** [comment]
>above responsem but louder and with more "nerd"

**Comment by Shapsy** [comment]
Cap's ID definitely shows just tested

**Comment by Iamgoofball** [comment]
its orange

File: code/game/objects/items/weapons/storage/wallets.dm [hunk 1]
```diff
@@ -57,22 +57,20 @@
 			refreshID()
 
 /obj/item/weapon/storage/wallet/update_icon()
-
+	icon_state = "wallet"
 	if(front_id)
```
Comment (KorPhaeron) [inline code/game/objects/items/weapons/storage/wallets.dm]: can't this all just be

if(front_id)
icon_state = "walletid_[front_id.icon_state]" or something

**Pull request merged**